pkg/3scale/amp/component/system.go

```diff
@@ -869,11 +929,25 @@ func (system *System) systemConfigVolumeMount() v1.VolumeMount {
 	}
 }
 
+func (system *System) s3CredsProjectedVolumeMount() v1.VolumeMount {
+	tokenMountPath := strings.TrimSuffix(system.Options.S3FileStorageOptions.TokenPath, TokenName)
```

Comment:
It  would be better if it was parsed and processed (removing the file name from the path) using file [filepath](https://pkg.go.dev/path/filepath) lib

```go
package main

import (
	"fmt"
	"path/filepath"
)

func main() {
	fmt.Println("Mount path: " + filepath.Dir("/var/run/secrets/openshift/serviceaccount/token"))
	fmt.Println("Token file: " + filepath.Base("/var/run/secrets/openshift/serviceaccount/token"))
}
```

Reply:
done